packages/notifier/test/test-notifier-examples.js

```diff
@@ -35,6 +35,7 @@ test('notifier for-await-of cannot eat promise', async t => {
   const { updater, notifier } = makeNotifierKit();
   paula(updater);
   const subP = Promise.resolve(notifier);
+  // @ts-ignore We are testing a case which violates the static types
```

Comment:
This type error only showed up in vscode. `yarn lint` was not complaining. Why?

Reply:
> This type error only showed up in vscode. `yarn lint` was not complaining. Why?

The only way this can happen is with version skew between VSCode and the `typescript` NPM package.  Can you compare VSCode's Cmd-Shift-p "TypeScript: Select TypeScript Version..." with `grep '"typescript"' agoric-sdk/package.json` ?